config/examples/backend.yml

```diff
@@ -1,18 +1,25 @@
+base: &default
+  url: <%= ENV.fetch('BACKEND_URL', 'http://localhost:3001') %>
```

Comment:
We need to talk about this... 

Currently, there is BACKEND_ROUTE env var, which is used by 3scale-operator. But I think this is confusing, as has "route" in the name, but in reality the value represents the **service URL** in OpenShift with "/internal/" suffix.

There is also THREESCALE_CORE_INTERNAL_API in core.yml which is supposed to hold the same value.

So, the proposal is to have BACKEND_URL (or maybe BACKEND_INTERNAL_URL to be more specific) to refer to the internal backend URL (service URL in OpenShift), and BACKEND_PUBLIC_URL that would be referring to the OpenShift route, that would be a URL accessible to the end users (via the browser).